### DISCUSSION TITLE
search by alert name (unified alerting)

### DISCUSSION THREAD

**Discussion by jeffqumata** [grafana/grafana#40472] [Alerting]
I'm sending alerts to slack via alertmanager and at the moment there seems to be no easy way to generate a direct link to take the user to the alert (and graph) in grafana which generated the slack message.

I've seen there is a view created by this [PR](https://github.com/grafana/grafana/pull/35248) which shows the data I need but generating the link URL from outside grafana seems non-trivial, specifically the data required for the `:ruleIdentifier` isnt available in the context of alertmanager. 

I note that there *is* a 'Search by label' filter but it doesnt seem to be able to filter on the name of the alert, only explicit labels. Is there a "hidden" label or other means to filter the alerts by name? (I tried `{__name=foo}` as a filter on the offchance it was similar to prom metrics but no luck). Or if there is an alternative `:ruleIdentifier` which im not aware of?

**Comment by shotarok** [comment]
Recently we started using `grafana/grafana:9.1.6`, and I faced this issue. It turns out our alerts have the names with the label: `alertname`. Therefore, alabel filtering: `alertname=~<regex>` worked for me.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
